sql/src/test/java/org/apache/druid/sql/calcite/schema/SegmentMetadataCacheTest.java

```diff
@@ -1447,6 +1456,25 @@ public void testStaleDatasourceRefresh() throws IOException, InterruptedExceptio
     Assert.assertNull(schema.getDatasource("wat"));
   }
 
+  @Test
+  public void testRefreshShouldEmitMetrics() throws InterruptedException
+  {
+    StubServiceEmitter emitter = new StubServiceEmitter("broker", "host");
+    buildSchemaMarkAndTableLatch(SEGMENT_CACHE_CONFIG_DEFAULT, emitter);
```

Comment:
Instead of this, can we just instantiate a `SchemaMetadataCache` and call `refresh` on it?